Rocky.class.nut

```diff
@@ -20,10 +30,30 @@ class Rocky {
 
     constructor(settings = {}) {
         // Initialize settings
-        if ("timeout" in settings) _timeout = settings.timeout;
-        if ("allowUnsecure" in settings) _allowUnsecure = settings.allowUnsecure;
-        if ("strictRouting" in settings) _strictRouting = settings.strictRouting;
-        if ("accessControl" in settings) _accessControl = settings.accessControl;
+        if ("timeout" in settings) {
```

Comment:
To be honest I don't feel comfortable with leaving these checks in the code.

There seems to be some miscommunication, as my understanding is that we agreed to not introduce them into the code and address in the documentation.

The bad thing is that it blows up the code size and impact performance (not sure how critical for Rocky use cases though).

Reply:
I'm happy to remove them.